src/integrationTest/resources/application.properties

```diff
@@ -14,6 +14,7 @@ storage.url=http://localhost
 storage.blob_copy_timeout_in_millis=30000
 storage.blob_copy_polling_delay_in_millis=300
 storage.blob_lease_timeout=60
+storage.proxy_enabled=false
```

Comment:
This is false by default do we need this ?

Reply:
I tested in local after removing this property. the integration tests are running fine